src/components/AuthPage/ForgotPassword/index.jsx

```diff
@@ -16,151 +16,161 @@ import { Alert } from "@mui/material";
 import PropTypes from "prop-types";
 
 const useStyles = makeStyles({
-	root: {
-		padding: "2rem",
-		background: "theme.palette.background.paper",
-		border: "none",
-		boxShadow: "none",
-	},
-	heading: {
-		fontWeight: 600,
-	},
+  root: {
+    padding: "2rem",
+    background: "theme.palette.background.paper",
+    border: "none",
+    boxShadow: "none"
+  },
+  heading: {
+    fontWeight: 600
+  }
 });
 
 const ForgotPassword = ({
-	rootBackground = "rgba(0,0,0,.01)",
-	confirmationText = "We have sent you an email containing the link to reset your password .Please check your inbox including spams",
-	fontweight = "800",
-	buttonColor = "blue",
+  rootBackground = "rgba(0,0,0,.01)",
+  confirmationText = "We have sent you an email containing the link to reset your password .Please check your inbox including spams",
+  fontweight = "800",
+  buttonColor = "blue"
 }) => {
-	const firebase = useFirebase();
-	const [loading, setLoading] = useState(false);
-	const [error, setError] = useState("");
-	const [success, setSuccess] = useState(false);
-	const [email, setEmail] = useState("");
-	const [open, setOpen] = React.useState(true);
-	const errorProps = useSelector(({ auth }) => auth.profile.error);
-	const loadingProps = useSelector(({ auth }) => auth.profile.loading);
-	const dispatch = useDispatch();
+  const firebase = useFirebase();
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState("");
+  const [success, setSuccess] = useState(false);
+  const [email, setEmail] = useState("");
+  const [open, setOpen] = React.useState(true);
```

Comment:
Can we use `useState` as similar to what we did for above other useStates? This is creating code inconsistency I feel, what's your input?